### PR TITLE
Add 'make preflight' command to make it easier to keep local installs ready for work

### PR DESCRIPTION
**Pull request by stevejalim**
## One-line summary

This changeset adds `make preflight` as a new Makefile command so it just takes one command to bring a local install up to latest state.

* installs latest Python deps
* installs latestJS deps
* gets latest DB and l10n files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -175,4 +176,8 @@ install-local-python-deps:
 install-local-docs-deps: install-local-python-deps
 	pip install -r requirements/docs.txt
 
-.PHONY: all clean build pull docs livedocs build-docs lint run stop kill run-shell shell test test-image rebuild build-ci test-ci fresh-data djshell run-prod run-pocket run-pocket-prod build-prod test-cdn compile-requirements check-requirements install-local-python-deps install-local-docs-deps
+preflight:
+	${MAKE} install-local-python-deps
+	$ ./manage.py migrate
+
```
Comment (stevejalim) [inline Makefile:182]: @alexgibson Do you think it'd be useful to automatically trigger `npm install` here, too?

File: package-lock.json [hunk 2]
```diff
@@ -1,9901 +1,8 @@
 {
   "name": "bedrock",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (alexgibson) [inline package-lock.json:4]: @stevejalim looks like you might be running an outdated version of NPM here as the lockfile has reverted to an older format?

Comment (stevejalim) [reply package-lock.json]: 🤦 thanks -- fixed!

**Review by alexgibson — APPROVED** [review]
r+, nice work!

**Pull request merged**